src/pg/pg-store.ts

```diff
@@ -266,6 +270,16 @@ export class PgStore extends BasePgStore {
     return result[0].max.toString();
   }
 
+  async getInscriptionsPerBlockETag(): Promise<string> {
+    const result = await this.sql<{ block_hash: string }[]>`
+      SELECT block_hash
```

Comment:
I think we should append the count here too, just in case we *partially* ingest a block for any reason and we need to replay that block again (making the hash stay the same but the inscription count be different). You could query the count here and then below do:

```typescript
return `${result[0].block_hash}:${result[0].count}`;
```

Just make sure you handle cases where the `inscriptions_per_block` table is empty